### PR TITLE
Update digitalmarketplace-utils from 52.2.0 to 52.6.0

### PR DESCRIPTION
**Pull request by lfdebrux**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 -e .
 # DON'T FORGET TO UPDATE setup.py !!
```
Comment (katstevens) [inline requirements.txt:2]: I think we forgot to update `setup.py` 😹

Comment (lfdebrux) [reply requirements.txt:2]: 😅 woops 🤦

Comment (lfdebrux) [reply requirements.txt:2]: On the other hand, the library still works with version 52.2.0 of dmutils, it's just the tests that change, so maybe the version in setup.py could stay the same? What do you think?

Comment (katstevens) [reply requirements.txt:2]: I'd rather keep them consistent if we can.

**Pull request merged**